pkg/workceptor/command.go

```diff
@@ -379,40 +400,63 @@ func filenameExists(filename string) error {
 	return nil
 }
 
-func (cfg signingKeyPrivateCfg) Prepare() error {
+func (cfg SigningKeyPrivateCfg) Prepare() error {
+	duration, err := cfg.PrepareSigningKeyPrivateCfg()
+
+	if err != nil {
+		return fmt.Errorf(err.Error())
+	}
+
+	MainInstance.SigningExpiration = *duration
```

Comment:
I might be misunderstanding this, but here is what I think is happening.
Main Instance is the global netceptor instance.
Assigning the keys to the global instance will allow all instances to use it, which is fine, however, will we ever have a case where we have multiple receptor instances, some signed and others unsigned?

And if yes, should netceptor be passed into this function instead? `Prepare(n *netceptor)`

Reply:
Isn't `Prepare` called by the `cmdline` package? I don't think we can influence which arguments this is called with. But I reserve the right to be wrong. 🙂

Reply:
I get you! 
Happy to resolve this.